styles.css

```diff
@@ -0,0 +1,520 @@
+* {
+	box-sizing: border-box;
+}
+
+html {
+	font-size: 16px;
+}
+
+body {
+	font-family: 'Gelasio', serif;
+	color: #0E1116;
+	background-color: #374B68;
+}
+
+header {
+	background-color: #374B68;
+	padding-top: 16px;
+	height: 160px;
+	width: 100%;
+	position: fixed;
+	top: 0;
+	z-index: 1;
+}
+
+.header-top {
+	text-align: center;
+	overflow: hidden;
+}
+
+.logo {
+	display: inline-block;
+	height: 80px;
+	margin-right: 30px;
+}
+
+#burger-menu {
+	display: none;
+}
+
+h1 {
+	display: inline-block;
+	padding: 0;
+	margin: 0;
+	font-size: 65px;
+	font-weight: 400;
+}
+
+h2 {
+	font-size: 40px;
+	margin: 0;
+	padding: 20px 0;
+	height: 80px;
+	text-transform: uppercase;
+	font-weight: 400;
+	color: #C0D7E0;
+}
+
+h3 {
+	display: block;
+	padding: 0;
+	margin: 0;
+	font-size: 30px;
+	font-weight: 400;
+}
+
+h4 {
+	margin: 0;
+	padding: 0;
+	font-size: 28px;
+}
+
+.header-top h1 {
+	position: relative;
+	bottom: 17px;
+	color: #20385E;
+}
+
+.accent-color {
+	color: #C0D7E0;
+}
+
+.accent-largetext {
+	font-size: 38px;
+	font-weight: 500;
+}
+
+.accent-serif {
+	font-family: 'Gelasio', serif;
+}
+
+nav {
+	background-color: #C0D7E0;
+	padding: 10px 0;
+	overflow: auto;
+}
+
+a {
+	display: inline-block;
+	text-decoration: none;
+	color: #0E1116;
+}
+
+.nav-links {
+	overflow: auto;
+	width: 800px;
+	margin: 0 auto;
+	padding: 0 30px;
+	font-size: 20px;
+	font-family: 'Poppins', sans-serif;
+	text-transform: uppercase;
+}
+
+nav ul {
+	list-style-type: none;
+	margin: 0;
+	padding: 0;
+	overflow: hidden;
```

Comment:
check this is doing something, if not remove it

Reply:
Overflow-hidden is stopping the scroll bar white box from appearing